pkg/kubelet/kubelet.go

```diff
@@ -339,6 +399,32 @@ func (kl *Kubelet) setupDataDirs() error {
 	return nil
 }
 
+func (kl *Kubelet) setupTmpfsDataDirs() error {
+	kl.tmpfsRootDirectory = path.Clean(kl.tmpfsRootDirectory)
+	// TODO: handle edge cases like mount already exists
```

Comment:
@thockin 

> If the tmpfs is a function of the root directory we can't blindly unmount/remount it - it will have been bind-mounted into pods, so either we won't be allowed to unmount it or we will totally screw over any running pods. This is important - kubelet MUST be able to crash and restart without killing pods.

I think this holds no matter what the path of the tmpfs root is, and therefore the best thing to do when the mountpoint already exists is to just skip the mount attempt.  Considering it to be an error isn't an option - the mount point already existing is expected.  Eventually the kubelet will crash or go down for maintenance, and when it comes back up, the mountpoint will already exist.